pyproject.toml

```diff
@@ -21,7 +21,7 @@ dynamic = ["version"]
 dependencies = [
     "mqt.bench>=0.4.4,<0.4.5",
     "sb3_contrib>=2.0.0",
-    "scikit-learn>=1.0.2",
+    "scikit-learn>=1.3.0, <1.3.1",
```

Comment:
Why the rather strict requirement here? Do we need any new feature of scikit-learn?

Reply:
No, but v1.3 crashes when using the v1.2.2 pre-trained model.

Reply:
So we need updated pre-trained models for this, right? Maybe I am misinterpreting your comment.

Reply:
Yes, but only updating the pre-trained classifier would lead to situations where users with a version below v1.3 try to use a trained model based on v1.3 which would most likely lead to errors.

Reply:
Was kind of a misunderstanding on my end. I did not see the updated ML model in the diff! Then I am totally fine with this change 👍🏼 Feel free to resolve.